README.md

```diff
@@ -0,0 +1,59 @@
+# CHBH-on-BEAR Documentation
+
+This documentation covers aspect of running CHBH-on-BEAR workflows on the University of Birmingham's BlueBEAR HPC system.
+
+## Publishing
+
+The documentation is automatically built and published to <https://ajquinn.github.io/chbh-on-bear> on every push to the `main` branch of this repository.
+
+This is taken care of by the [`deploy` GitHub Actions workflow](https://github.com/AJQuinn/chbh-on-bear/tree/main/.github/workflows/deploy.yml).
+
+## Using `mkdocs`
+
+### Creating a Development Environment and Installing `mkdocs`
+
+To install all the required `mkdocs` Python packages, use the provided [requirements.txt](https://github.com/AJQuinn/chbh-on-bear/tree/main/requirements.txt) file. The recommendation for development is to do this inside a dedicated virtual environment:
+
+```shell
+git clone https://github.com/AJQuinn/chbh-on-bear
+cd chbh-on-bear
+python -m venv ./.venv
+source ./venv/bin/activate
```

Comment:
Should this line also include the `.` in the  venv dir?

`source ./.venv/bin/activate`

Not sure which is generally recommended.

Reply:
Good spot!

```suggestion
source ./.venv/bin/activate
```